packages/block-editor/src/components/spacing-sizes-control/style.scss

```
@@ -0,0 +1,129 @@
+.tools-panel-item-spacing {
+	display: grid;
+	grid-template-columns: auto 1fr auto;
+	grid-row-gap: $grid-unit-05;
+}
+
+.component-spacing-sizes-control {
+	box-sizing: border-box;
+	padding-bottom: 12px;
+	width: 100%;
+	display: contents;
+
+	legend {
+		padding-left: 0;
```

Browser user agents for Chrome, Safari etc add both left and right padding to `legend` elements. Is it deliberate to combine the right padding still with the margin-left on the hint?
```suggestion
		padding: 0;
```

---

nothing deliberate there, gone now as not really needed.